File: samples/snippets/csharp/VS_Snippets_CLR_System/system.Char.GetUnicodeCategory/CS/getunicodecategory.cs
// <snippet1>
using System;

public class GetUnicodeCategorySample {
	public static void Main() {
		char ch2 = '2';
		string str = "Upper Case";

		Console.WriteLine(char.GetUnicodeCategory('a'));		// Output: "LowercaseLetter"
		Console.WriteLine(char.GetUnicodeCategory(ch2));		// Output: "DecimalDigitNumber"
		Console.WriteLine(char.GetUnicodeCategory(str, 6));		// Output: "UppercaseLetter"
	}
}
// </snippet1>

File: snippets/fsharp/VS_Snippets_CLR_System/system.Char.GetUnicodeCategory/FS/getunicodecategory.fs
// <snippet1>
open System

let ch2 = '2'
let str = "Upper Case"

printfn $"{Char.GetUnicodeCategory 'a'}"        // Output: "LowercaseLetter"
printfn $"{Char.GetUnicodeCategory ch2}"        // Output: "DecimalDigitNumber"
printfn $"{Char.GetUnicodeCategory(str, 6)}"    // Output: "UppercaseLetter"
// </snippet1>